packages/mobile/src/components/share-drawer/useShareToStory.tsx

```diff
@@ -320,9 +351,13 @@ export const useShareToStory = ({
         let session: FFmpegSession
         const SHOWFREQS_SEGMENT =
           'aformat=channel_layouts=mono:sample_rates=16000,adynamicsmooth,showfreqs=s=900x40:fscale=log:colors=#ffffff70'
+        const thirdLayerSegment =
+          platform === 'tiktok'
+            ? '[vid];[1:v]scale=396:548[stkr];[vid][stkr]overlay=format=auto:x=72:y=200;'
+            : ';'
         try {
           session = await FFmpegKit.execute(
-            `${audioStartOffsetConfig}-i ${streamMp3Url} -filter_complex "${backgroundSegment}[bg];[0:a]${SHOWFREQS_SEGMENT}[fg];[0:a]${SHOWFREQS_SEGMENT},vflip[fgflip];[bg][fg]overlay=format=auto:x=-100:y=H-h-100[fo];[fo][fgflip]overlay=format=auto:x=-100:y=H-h-60;[0:a]anull" -pix_fmt yuv420p -c:v libx264 -preset ultrafast -c:a aac -t 10 ${storyVideoPath}`
+            `${audioStartOffsetConfig}-i ${streamMp3Url} -i ${stickerUri} -filter_complex "${backgroundSegment}[bg];[0:a]${SHOWFREQS_SEGMENT}[fg];[0:a]${SHOWFREQS_SEGMENT},vflip[fgflip];[bg][fg]overlay=format=auto:x=-100:y=H-h-100[fo];[fo][fgflip]overlay=format=auto:x=-100:y=H-h-60${thirdLayerSegment}[0:a]anull" -pix_fmt yuv420p -c:v libx264 -preset ultrafast -c:a aac -t 10 ${storyVideoPath}`
```

Comment:
Should this stickerUri always be in the ffmpeg command? Or only for the tiktok case?

Reply:
ooh good catch, just for tiktok